### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by MrHug**
I've added some pages about how you make contributions. Let me know what you think.

NB: I still have to proofread this myself, but I have to leave now, so will do so tonight :wink:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,26 @@
-# subtitles
-All subtitles for the EX101x MOOC can be found here
+# Subtitles
+All subtitles for the EX101x MOOC can be found here.
+
+## Adding a translation to this repository
+If you are interested in adding a translation to this repository, please follow
+the steps below. If you are interested in proposing a fix to existing subtitles,
+please check the section on [fixes](./submittingFixes.md).
```
Comment (levaja) [inline README.md]: SubmittingFixes.md is missing.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,26 @@
...
+To add a new translation to this repository, two situations are possible. Either
+we have already created branches for you to fill with the subtitles, or no such
+branches exist yet. If you are unsure whether we have created them, you can
+check by clicking on the dropdown menu that says "branch: master". You will get
+options like this:
+
+![Existing branches in the repository](./uploadInstructions/figures/branchswitch2.png)
+
+If there is a branch called "WeekXYY" where X is the week number and YY is the
+abbreviation of the language you want to add (e.g. NL for Dutch) than a branch
```
Comment (levaja) [inline README.md]: "then" instead of "than"

File: README.md [hunk 3]
```diff
@@ -1,2 +1,26 @@
...
+already exists and you can [add your work
+there](./uploadInstructions/existingBranch.md) If no such branch exists, you
+will have to [create one yourself](./uploadInstructions/newBranch.md). If you
+are not a part of our organisation you will have to [create a
+fork](./uploadInstructions/fork.md) before following the instructions above in
```
Comment (levaja) [inline README.md:23]: fork.md is missing.

File: uploadInstructions/existingBranch.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+Add a "commit message" telling us what you have added:
+
+![Add a commit message](figures/upload4.png)
+
+Click "Commit new file":
+
+![Commit new file](figures/upload5.png)
+
+Congratsulations your changes have now been committed to this branch! In order
```
Comment (levaja) [inline uploadInstructions/existingBranch.md]: Add comma after "Congratulations"

Comment (levaja) [inline uploadInstructions/existingBranch.md]: And fix to "Congratulations" (you have one 's' extra)

File: uploadInstructions/newBranch.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+Be sure to click: "Create a new branch for this change and open a new pull request" and fill in a sensible name for the branch (for instance: Week3LA) :
+
+![Commit new file](figures/upload6.png)
+
+Click "Commit new file":
+
+![Commit new file](figures/upload5.png)
+
+Congratsulations your changes have now been committed to a new branch! Future
```
Comment (levaja) [inline uploadInstructions/newBranch.md]: "Congratsulations" -> "Congratulations, "

File: uploadInstructions/newBranch.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+additions in the same topic can be done [in the same branch](./sameBranch.md).
```
Comment (levaja) [inline uploadInstructions/newBranch.md]: sameBranch.md is missing

File: uploadInstructions/pullRequest.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
+# Creating a Pull Request (PR)
+
+In order to have your changes merged into our `master` branch, you will have to
+submit a pull request. We will then review your changes or new subtitles and
+give you some feedback. You might have to [make some
+changes](./submittingFixes.md) before we accept your pull request.
```
Comment (levaja) [inline uploadInstructions/pullRequest.md:6]: submittingFixes.md is missing.

**Comment by levaja** [comment]
Looks good, I've pointed out some minor issues, I guess you are aware of the missing files, but I've just mentioned them in line comments :)

**Comment by MrHug** [comment]
Thanks a bunch! I was aware of this, and will be adding them soon ;)

**Comment by MrHug** [comment]
@MOOCDataAnalysis/owners Ready for review and merger.

**Comment by MrHug** [comment]
https://github.com/MOOCDataAnalysis/subtitles/blob/ReadmeUpdate/uploadInstructions/fork.md 

Seems to be working?

**Comment by levaja** [comment]
Yeah, sorry, looked into wrong commit, that's why I've deleted my comment :)

Everything else looks good, great job! :+1:

**Comment by MrHug** [comment]
Merge time?

**Comment by levaja** [comment]
Indeed!

**Pull request merged**